### ISSUE TITLE
Seeding - Add support for type specification

### ISSUE THREAD

**Issue by cdelgadob** [99xt/serverless-dynamodb-local#133]
**Actual Behaviour**

When defining the seed data, you can't specify whether you need an array or a set.

**Expected Behaviour**

Although this should keep working as it is, it should also allow to define the type of the field.

Define an attribute with all elements of the same type, it will be converted to a list.

- This could either support the DynamoDB type notation: {"SS": ["foo", "bar"]} for a StringSet.

- Or could set up a flag per table to indicate if the objects should be marshaled or not, using for instance, this library: [https://www.npmjs.com/package/dynamodb-marshaler](https://www.npmjs.com/package/dynamodb-marshaler)

**Steps to reproduce it**

NA

**LogCat for the issue**

NA
**Screenshots of the issue**

NA

**Would you like to work on the issue?**

I'm afraid I'm not enough skilled for this :S

**Comment by marksteele** [comment]
+1
Also doesn't work if your value is a number, only works with strings or objects.

This works:
`[{"id": "foo","length":"1"},{"id": "bar","length":"7"}]`

This doesn't:
`[{"id": "foo","length":1},{"id": "bar","length":7}]`

**Comment by rajikaimal** [comment]
@AshanFernando friendly ping 😄

**Comment by AshanFernando** [comment]
@marksteele @rajikaimal I just checked the seeder.js and its currently using DynamoDBDocument Client which don't expect the data type. However, it seems like the numbers don't support because the line number 115 in seeder.js users require(location) to load the seed file which will expect in JSON format. 

My guess is if we can use JSON.parse(fs.readFileSync(location, 'utf8')); it should work for numbers like "length": 7.

@marksteele @rajikaimal  Will you be able to replace this line in your code inside node-modules/serverless-dynamodb-local/src/seeder.js?

**Comment by marksteele** [comment]
@rajikaimal @AshanFernando 

Here's the fix to get numeric values from the JSON to work, had to change the unmarshalBuffer to guard against undefined value in the loop.

```
/**
 * Transform all selerialized Buffer value in a Buffer value inside a json object
 *
 * @param {json} json with serialized Buffer value.
 * @return {json} json with Buffer object.
 */
function unmarshalBuffer(json) {
  _.forEach(json, function(value, key) {
    if (value !== undefined && value.type==="Buffer") {
      json[key]= new Buffer(value.data);
    }
  });
  return json;
}
```

**Comment by vkarponen** [comment]
Some of the DynamoDb functionality like `updateItem({...ADD...})` require a **Set** instead of a **List**. Being unable to seed a Set is very inconvenient.

**Comment by vkarponen** [comment]
This is now released in 0.2.30. Please close the issue.

**Comment by rehrumesh** [comment]
Thanks, @vkarponen for the implementation.
